src/yunohost/settings.py

```diff
@@ -278,10 +283,12 @@ def trigger_post_change_hook(setting_name, old_value, new_value):
 #
 # ===========================================
 
+@post_change_hook("security.ciphers.compatibility")
+def reconfigure_nginx(setting_name, old_value, new_value):
+    if old_value != new_value:
+        service_regen_conf(names=['nginx'], force=True)
 
-#@post_change_hook("example.int")
-#def myfunc(setting_name, old_value, new_value):
-#    print("In hook")
-#    print(setting_name)
-#    print(old_value)
-#    print(new_value)
+@post_change_hook("service.ssh.ciphers.compatibility")
+def reconfigure_ssh(setting_name, old_value, new_value):
+    if old_value != new_value:
+        service_regen_conf(names=['ssh'], force=True)
```

Comment:
Note that you should *not* use `force=True` if you don't have a good reason to do it... otherwise that will overwrite manually-edited conf which might have desastrous consequences. If the user manually edited in a conf file, then a message will be displayed accordingly, and it's his/her responsability to handle the file change. (We hope someday™ to have a smarter things such as merging manual changes and yunohost changes with a git-like merge, or user being able to create custom patch, but for now that's not the case)

Reply:
It was intended. I fixed it.